package.json

```diff
@@ -79,6 +79,8 @@
     "@testing-library/jest-dom": "^5.1.0",
     "@testing-library/react": "^9.4.0",
     "@testing-library/react-hooks": "^3.2.1",
+    "@types/react": "^16.8.6",
+    "@types/react-dom": "^16.8.6",
```

Comment:
I had to add a dev dependency on `@types/react` to get typechecking to work in our `.tsx` stories since React doesn't bundle their own types